.github/workflows/publish-openapi.yaml

```diff
@@ -1,19 +1,14 @@
 name: OpenAPI Publisher
 
 on:
-  push:
-    branches:
-      - main
-    paths:
-      - openapi/**
+  workflow_run:
+    workflows: ['OpenAPI Validator']
+    branches: [main]
+    types:
+      - completed
```

Comment:
do we need the `types` here?

Reply:
>By default, all activity types trigger workflows that run on this event. 

I think excluding would cause the publisher to run without the validator necessarily succeeding (via `requested` or `in_progress` types)

Reply:
Would it be good to add an `on-success` flag?
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-a-workflow-based-on-the-conclusion-of-another-workflow

Reply:
Otherwise, if I'm understanding the docs correctly, we would still publish if the validator failed. honestly would have made sense to have a `type: succeeded` but 🤷